### PR TITLE
[Merged by Bors] - Enable Log Aggregation for Superset

### LINKED ISSUE #317: Enable Log Aggregation for Superset
This is part of stackabletech/issues#288.

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Add log aggregation

Closes #317 

## Integration tests

[Build Status](https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/superset-operator-it-custom/28/)

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: rust/crd/src/supersetdb.rs [hunk 1]
```diff
@@ -1,27 +1,77 @@
...
+    EnumIter,
+    JsonSchema,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    Serialize,
+)]
+#[serde(rename_all = "kebab-case")]
+#[strum(serialize_all = "kebab-case")]
+pub enum Container {
```
Comment (maltesander) [inline rust/crd/src/supersetdb.rs:43]: Just nitpicking: Can we rename this to something like `InitDbContainer`?

**Review by maltesander — APPROVED** [review]
Works! LGTM.

File: rust/crd/src/supersetdb.rs (hunk 1 above)
Comment (siegfriedweber) [reply rust/crd/src/supersetdb.rs]: renamed in 2522efa80079f7ff22998f623dca26adc9245762

**Comment by siegfriedweber** [comment]
bors merge

**Pull request closed without merging**